create-env.bat

```diff
@@ -104,7 +104,7 @@ REM
     ECHO The scripts folder at !CONDASCRIPTS! has been detected as a valid conda installation.
     ECHO The conda commands from this directory are used in the following.
 
-    CALL !CONDASCRIPTS!activate base && (
+    CALL !CONDASCRIPTS! activate base && (
```

Comment:
This is not a fix. I expect `!CONDASCRIPTS!activate` to evaluate to e.g. `C:\Users\YOUR_USERNAME\Anaconda3\Scripts\activate`. A space between the folder and the string `activate` would break it. Furthermore, this batch file works perfectly on my computer. Could you provide the output that shows how the syntax error (which is none on my side) affects you?

Reply:
On my computer the resulting string is `C:\PathToConda\anaconda3\Library\bin\conda.batactivate` which is not a valid command. I think this happens because the folder `C:\PathToConda\anaconda3\Library\bin\` is an entry in my PATH variable as well and comes before `C\PathToConda\anaconda3\Scripts\`, so this is found first when a valid installation is searched in the PATH environment variable.

Reply:
Ah sure, then the issue lies here: https://github.com/1kastner/conflowgen/blob/main/create-env.bat#L64-L65

The where command was used as if it was returning the directory but instead it (obviously) returned the path to the conda command itself. I guess adding `..` is fine, as `CALL C:\Users\YOUR_USERNAME\Anaconda3\Scripts\conda.exe\..\activate.bat conflowgen` works fine of the CMD for me.

So would you mind undoing this part of the pull request and instead tweak the line at https://github.com/1kastner/conflowgen/blob/main/create-env.bat#L67?

Reply:
I added a new workflow to check for the conda installation which at least covers the case that has worked so far and which should not break. It could also be great to have one workflow that covers your case but I have not found the time to do that yet.